include/language-factory.php

```diff
@@ -94,6 +95,8 @@ public static function get_from_terms( array $terms ) {
 			}
 		}
 
+		$data['is_default'] = $options['default_lang'] === $data['slug'];
```

Comment:
For you, it's clearer to set the property than when we initialize the `$data` array at the beginning of the method?